bakerydemo/breads/wagtail_hooks.py

```diff
@@ -0,0 +1,45 @@
+from wagtail.snippets.models import register_snippet
+from wagtail.snippets.views.snippets import SnippetViewSet, SnippetViewSetGroup
+
+from bakerydemo.breads.models import BreadIngredient, BreadType, Country
+
+
+class BreadIngredientSnippetViewSet(SnippetViewSet):
+    model = BreadIngredient
+    ordering = ("name",)
+    search_fields = ("name",)
```

Comment:
This is missing inspect view from the original `ModelAdmin` config

```suggestion
class BreadIngredientSnippetViewSet(SnippetViewSet):
    model = BreadIngredient
    ordering = ("name",)
    search_fields = ("name",)
    inspect_view_enabled = True
```

Reply:
Thanks!